app/src/pages/common/settingsPage/defectTypesTab/defectTypePropType.js

```diff
@@ -0,0 +1,10 @@
+import PropTypes from 'prop-types';
+
+export const defectTypePropType = {
```

Comment:
I think it's better to write `defectTypePropType = PropTypes.shape(...)` here and remove the same construction in the components where it was used.